### ISSUE TITLE
Warning when rendering images

### ISSUE THREAD

**Issue by sapegin** [alexkuz/markdown-react-js#5]
```
Warning: img is a void element tag and must not have `children` or use `props.dangerouslySetInnerHTML`. Check the render method of MDReactComponent.
```

Images have `children = []`;

**Comment by davnicwil** [comment]
this is a warning introduced by react 0.14.x - it complains when you pass anything at all as children in the React.createElement call. So you have to explicitly not, for that tag (and possibly others, that's the only one flagged in the tests).

As it's not passing children, I think the warning can basically be ignored, but I've fixed it anyway in my fork @ https://github.com/davnicwil/markdown-react-js/tree/1.x

This branch basically upgrades the entire component to be totally up to date with 0.14.x.

**Comment by kof** [comment]
duplicate of #13, should be fixed, please reopen #13 if not
